### PR TITLE
SCMOD-15060: Update copyright year

### PR DESCRIPTION
**Pull request by xbreizh**
https://portal.digitalsafe.net/browse/SCMOD-15060

### CODE REVIEW

**Comment by xbreizh** [comment]
@dermot-hardy Sorry I forgot about the Copyright

File: pom.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
     </scm>
 
     <properties>
-        <copyrightYear>2020</copyrightYear>
+        <copyrightYear>2021</copyrightYear>
         <copyrightNotice>Copyright ${copyrightYear} Micro Focus or one of its affiliates.</copyrightNotice>
```
Comment (dermot-hardy) [inline pom.xml]: Drop this line that is overriding the `copyrightNotice` property.  It should only be there when the inception year and the copyright year are the same year.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (xbreizh) [reply pom.xml]: Done

Comment (dermot-hardy) [reply pom.xml:61]: You've dropped the wrong line.

Comment (dermot-hardy) [reply pom.xml]: You've dropped the wrong line.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (xbreizh) [reply pom.xml]: restored/dropped

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
I created a developer build and it fails.

```
[ERROR] Failed to execute goal com.mycila:license-maven-plugin:3.0:check (check-license-headers) on project swagger-ui-dist: Some files do not have the expected license header -> [Help 1]
```

**Comment by xbreizh** [comment]
> I created a developer build and it fails.
> 
> ```
> [ERROR] Failed to execute goal com.mycila:license-maven-plugin:3.0:check (check-license-headers) on project swagger-ui-dist: Some files do not have the expected license header -> [Help 1]
> ```

Indeed. Build is now passing.

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**